src/main/java/tictactoe/Game.java

```diff
@@ -0,0 +1,20 @@
+package tictactoe;
+
+public class Game {
```

Comment:
Nice! 👏  Much smaller than yesterday's plus it's very clear what this class is responsible for. Well done!